src/main/java/com/uid2/shared/auth/EncryptionKeyAcl.java

```diff
@@ -17,4 +20,26 @@ public boolean canBeAccessedBySite(int siteId) {
 
     public boolean getIsWhitelist() { return isWhitelist; }
     public Set<Integer> getAccessList() { return accessList; }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Is there a reason why `isWhitelist` and `accessList` are both package-private and non-final? Can we make them private final?